apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -11,11 +11,15 @@ var dom = require('../dom');
 /**
  * Handles displaying tooltips on Droplet's ACE editor when filling in
  * an empty parameter.
+ * Also will augment ACE editor's Live autocomplete by invoking scoped
+ * autocomplete dropdowns for each parameter.
  * @param {DropletTooltipManager} dropletTooltipManager
  * @constructor
  */
 var DropletAutocompleteParameterTooltipManager = function (dropletTooltipManager) {
   this.dropletTooltipManager = dropletTooltipManager;
+  this.showParamDropdowns = dropletTooltipManager.dropletConfig.showParamDropdowns;
+  DEFAULT_TOOLTIP_CONFIG.position = this.showParamDropdowns ? 'top' : 'bottom';
```

Comment:
Since this now gets mutated on init it might be helpful to move the config into a property and initialize it all here, e.g. `this.tooltipsterConfig = { [...]`